cmd/openshift-install/create.go

```diff
@@ -128,3 +140,75 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 		return nil
 	}
 }
+
+// FIXME: pulling the kubeconfig and metadata out of the root
+// directory is a bit cludgy when we already have them in memory.
+func destroyBootstrap(ctx context.Context, directory string) (err error) {
+	config, err := clientcmd.BuildConfigFromFlags("", filepath.Join(directory, "auth", "kubeconfig"))
+	if err != nil {
+		return err
+	}
+
+	client, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	discovery := client.Discovery()
+
+	apiContext, cancel := context.WithTimeout(ctx, 30*time.Minute)
+	defer cancel()
+	wait.Until(func() {
+		version, err := discovery.ServerVersion()
```

Comment:
inline the discovery here.. ?
```go
version, err := client.Discovery().ServerVersion()
```

Reply:
> `version, err := client.Discovery().ServerVersion()`

Part of me didn't want to repeat that `Discovery()` call if we didn't need to.  But I don't expect us to get so many dropped connections and such to make repeat `Discovery()` calls a performance issue ;).  You prefer inline calls to the local variable?

Reply:
https://github.com/kubernetes/client-go/blob/05d4e2f0fd654cb913c78624346785247edc66f7/kubernetes/clientset.go#L437-L443

it just returns the internal discovery client, so i think inline would have made no performance effect. But I'm fine as-is too :).